sdk/src/client/utils.rs

```diff
@@ -192,3 +193,40 @@ impl Client {
         Ok((Self::tag_to_utf8(payload)?, Self::data_to_utf8(payload)?))
     }
 }
+
+/// A password wrapper that takes care of zeroing the memory when being dropped.
+#[derive(Clone, Eq, PartialEq, Serialize, Deserialize, Zeroize, ZeroizeOnDrop, derive_more::From)]
+pub struct Password(String);
+
+impl Password {
+    pub fn as_bytes(&self) -> &[u8] {
+        self.0.as_bytes()
+    }
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
These tests feel...pointless.

Reply:
I thought the same, testing some derivations...

Reply:
tbh ... I agree with you, but they at least they don't hurt anybody.

Reply:
removed the tests